packages/packherd-require/src/default-transpile-cache.ts

```diff
@@ -0,0 +1,24 @@
+import type { TranspileCache } from './types'
+
+/**
+ * In memory transpile cache that is used if none was provided to `packherd:require`.
+ */
+export class DefaultTranspileCache implements TranspileCache {
+  private readonly _cache: Map<string, string> = new Map()
+
+  get (fullPath: string, _skipStaleCheck?: boolean): string | undefined {
```

Comment:
Is `_skipStaleCheck` supposed to be used anywhere?
```suggestion
  get (fullPath: string): string | undefined {
```

Reply:
I went ahead and removed it.